paddle/fluid/framework/ir/mkldnn/mkldnn_inplace_pass_tester.cc

```diff
@@ -153,6 +172,15 @@ TEST(MKLDNNInplacePass, inplace_elementwise_add) {
   // Two elementwise_add mkl-dnn enabled op instances to be made inplace
   MKLDNNInplacePassTest().MainTest("elementwise_add", false, 1);
 }
+TEST(MKLDNNInplacePass, inplace_tanh) {
+  MKLDNNInplacePassTest().MainTest("tanh", false, 1);
+}
+
+TEST(MKLDNNInplacePass, inplace_leaky_relu) {
+  // Input of leaky_relu is used as output of subsequent gelu, so no inplace
+  // cannot be done
+  MKLDNNInplacePassTest().MainTest("leaky_relu", false, 0);
+}
```

Comment:
What about adding tests for `relu` and `gelu`?

Reply:
Gelu technically is capable of in-place , but PaddlePaddle does not provide InplaceInferer for Gelu (gelu is implemented separatly from other activations) . In other words , currently inplace pass will not work for GELU as inplaceinferer is not present , once in general code there is inplaceinferer for gelu it all starts to work. As for relu/leaky_relu itp. In our code it is the same implementation. So does not matter which activation I choose for UT. Those tests are more for testing certain graphs confguration.